lib/cdo/google_drive.rb

```diff
@@ -117,6 +117,15 @@ def update_sheet(data, document_key, sheet_name)
       worksheet.save
     end
 
+    # Returns an ACL object for a spreadsheet document
+    # @param [String] document_key
+    # @return [GoogleDrive::Acl] ACL object that contains an array of GoogleDrive::AclEntry
```

Comment:
nit: Does this actually return an "ACL object"?  Or does it just return an array?

Reply:
It returns a `GoogleDrive::Acl` object according to the [API doc](https://www.rubydoc.info/github/gimite/google-drive-ruby/GoogleDrive/File#acl-instance_method). I've verified manually using `acl.class`.